.python_env_SAMPLE

```diff
@@ -29,3 +29,6 @@ MYSQL_ROOT_PW=root
 MYSQL_NAME=v1
 MYSQL_USER=v1
 MYSQL_PW=v1
+
+# Password given to 'admin' user created by initial-data.sh script.
+WAGTAIL_ADMIN_PW=admin
```

Comment:
In the interest of simplification, how would you feel about deprecating the use of this environment variable? That would mean modifying this PR to remove this change, removing its definition in the `.env_SAMPLE` [here](https://github.com/cfpb/cfgov-refresh/blob/master/.env_SAMPLE#L28), and hardcoding in in the initial data script [here](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/scripts/initial_data.py#L29).

Benefits:

1. Currently, if you don't define the `WAGTAIL_ADMIN_PW` environment variable, the initial data script invokes `make_password(None)`, which returns a valid hash, but then the user is left unable to login. This means the script can potentially be run leaving behind a broken admin user.
2. We hardcode the admin username as `admin`; hardcoding the admin password as `admin` would only simplify and make it easier to use locally.
3. This would let us simplify the documentation. Currently the docs instruct users to [create a superuser](https://cfpb.github.io/cfgov-refresh/usage/#3-load-indexes-launch-site), which isn't necessary, and also [mention](https://cfpb.github.io/cfgov-refresh/installation/#load-initial-data-into-database) the `WAGTAIL_ADMIN_PW` environment variable. Instead we could simplify and say that if `initial-data.sh` is run, you'll have an `admin` / `admin` superuser.

Drawbacks:

1. Limited configurability around local admin user password?

Reply:
I like this idea and will update the PR accordingly. Thanks @chosak